app/views/shared/_batch.html.erb

```diff
@@ -1,99 +1,78 @@
 
-  <table class="sortable table">
-    <thead>
-      <tr>
-        <% if edit -%>
-          <th width="3%">Drag</th>
```

Comment:
We can use the whole row as a drag handle, so can remove this. In turn this also means we don't need to specify a custom index column for the sort functionality.